Bottom: 7a5ef851c7ffa131c02fb5dc6c2d168eb4f55c90
Top:    c1fdac80e6e50d930e643dfac0373c804d423ddb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-10 15:29:00 +0000

thermal.c: Add #warning pointing out where we might get a speedup








---

diff --git a/src/thermal.c b/src/thermal.c
index 445b97d..fa87f28 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -432,6 +432,7 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         arc = CirclePoly (pin->X, pin->Y, pin->Thickness / 2);
         /* create a thin ring */
         poly_Boolean_free (pa, arc, &m, PBO_SUB);
+#warning THIS WOULD BE FASTER IF WE JUST SUTRACTED A CROSS SHAPE
         /* fix me needs error checking */
         if (style == 2)
           {
